perceval/utils.py

```diff
@@ -271,3 +284,85 @@ def node_to_dict(node):
     d = node_to_dict(tree)
 
     return d
+
+
+def create_jwt_token(github_app_id, pem_path):
+    """Create JWT token given the GitHub App ID and the private key PEM file.
+    We need this token to authenticate as a GitHub App
+
+    :param github_app_id: GitHub APP ID
+    :param pem_path: private key PEM file path
```

Comment:
If this is going to be generic, it shouldn't contain references to github. Maybe it makes more sense to keep in under the GitHub backend module.